iroh/src/commands/sync.rs

```diff
@@ -40,6 +40,8 @@ pub enum DisplayContentMode {
     Content,
     /// Display the hash of the content.
     Hash,
+    /// Displays the full hash of the content.
+    HashFull,
```

Comment:
I think this should just change the behaviour of `Hash` instead of introducing a new mode. We could then adjust the description of `Auto` to "diplays a shortened content hash"

```rs
pub enum DisplayContentMode {
    /// Displays the content if small enough, otherwise it displays a shortened content hash.
    Auto,
    /// Display the content unconditionally.
    Content,
    /// Display the hash of the content.
    Hash,
}

Reply:
`Hash` is currently used in listings, so that won't work unless we want to show the content or the full hash there

Reply:
ah I see. then I think we should invert the meaning of "hash", and introduce `ShortHash` to keep the current listing behaviour. I think my biggest sticking point is "hash should be hash"

Reply:
done